src/seeds/server.ts

```diff
@@ -18,15 +21,23 @@ export const createSeedsAPIServer = () => {
   app.use(express.json());
   app.use(authMiddleware);
 
-  app.post(`${apiVersionPrefix}/seeds/`, async (req: AuthRequest, res) => {
+  app.post(`${apiVersionPrefix}/messages/`, async (req: AuthRequest, res) => {
     try {
-      validateSeed(req.body)
+      await validateMessage(req.body)
     } catch (e: any) {
       return res.status(422).send({ error: e.message });
     }
 
     try {
-      await persistSeed(req.body, req.signer)
+      restrictAccess(req.body, req.signer);
```

Comment:
Usually I'd agree, but for now I'm going to err on the side of YAGNI for now seeing as we're sticking with a single endpoint design:
https://github.com/spinamp/spindexer/pull/122#discussion_r1009265322

Reply:
yeah, motivation for this is that if we decentralize in future, different nodes will likely communicate by just sending messages to eachother rather than having to split messages across via HTTP URLS and API calls.